src/modules/jcms_migrate/jcms_migrate.install

```diff
@@ -1121,3 +1121,29 @@ function jcms_migrate_update_8128() {
   $executable = new MigrateExecutable($plugins['jcms_subjects_json'], new MigrateMessage());
   $executable->import();
 }
+
+/**
+ * Remove genomics-evolutionary-biology msa.
+ */
+function jcms_migrate_update_8129() {
+  // Assign any remaining content to genetics-genomics, there shouldn't be any
+  // in production.
+  drush_invoke('msa-switch', ['genomics-evolutionary-biology', 'genetics-genomics']);
```

Comment:
Running this command will ensure that content in `continuumtest` has no reference to the outgoing MSA.